tests/test_api_build.py

```diff
@@ -62,21 +62,6 @@ def describe_root(cwd=None):
     return tag
 
 
-def package_has_file(package_path, file_path):
```

Comment:
For the record, I asked @soapy1 to remove this.  It is a duplicate definition, also defined in conda_build/utils.py - and the definition there is better.